kentikapi/internal/api_payloads/device.go

```diff
@@ -325,21 +327,31 @@ func payloadToDeviceLabel(p deviceLabelPayload) (models.DeviceLabel, error) {
 // deviceSitePayload represents JSON Device.Site payload as it is transmitted from KentikAPI.
 // deviceSitePayload embeddedd under Device differs from regular SitePayload in that all fields are optional.
 type deviceSitePayload struct {
-	CompanyID *models.ID `json:"company_id,omitempty"`
-	ID        *models.ID `json:"id,omitempty"`
-	Latitude  *float64   `json:"lat,omitempty"`
-	Longitude *float64   `json:"lon,omitempty"`
-	SiteName  *string    `json:"site_name,omitempty"`
+	CompanyID *int     `json:"company_id,omitempty"`
```

Comment:
```suggestion
	CompanyID *StringAsInt     `json:"company_id,omitempty"`
```
Please use StringAInt, the same like for `DevicePayload.UseBGPDeviceID` to simplify the code.
The same for ID below.
The same for devicePlanPayload below

Reply:
Done